src/devtools/client/debugger/src/workers/parser/tests/fixtures/async.js

```diff
@@ -1,5 +1,5 @@
 async function foo() {
```

Comment:
we should not be changing files in `src/devtools/client/debugger/src/workers/parser/tests/fixtures`